bioconda_utils/bioconda_utils-requirements.txt

```diff
@@ -22,3 +22,4 @@ six=1.11.*
 alabaster=0.7.*
 git=2.14.*
 conda-forge-pinning=2018.05.22
+python>=3.6
```

Comment:
For some reason one of the CI builds picked up `python 3.5.*` instead of `3.6`.
That one choked on https://github.com/bioconda/bioconda-utils/blob/41f39c06607fe142d47e5ba244cd88444e5367cd/bioconda_utils/utils.py#L217 complaining that `os.path.join` doesn't like the `*Path` objects.